### DISCUSSION TITLE
create table in malloy with DuckDB

### DISCUSSION THREAD

**Discussion by djouallah** [malloydata/malloy#625] [Q&A] [answered]
this does not seems to work, it is useful to create reproducible code. 

```
sql: my_sql_query is ||
  CREATE TABLE people(id INTEGER, name VARCHAR);
  INSERT INTO people VALUES (1, 'john'), (2, 'paul')
;;
```

**Comment by lloydtabb** [comment, marked as answer]
There are a number of ways of doing this.  See the example below for some test cases...

https://github.com/looker-open-source/malloy/blob/main/packages/malloy-db-test/src/nomodel.spec.ts#L412
